velox/common/caching/SsdFile.cpp

```diff
@@ -556,6 +556,7 @@ void SsdFile::clear() {
   std::fill(erasedRegionSizes_.begin(), erasedRegionSizes_.end(), 0);
```

Comment:
clear() seems to be only used by test and can we change the corresponding method in SSDCache and SSFile to testingClear() to mark them as test methods? Thanks!